[2023-04-08T23:11:35.541+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: api_dag.create_table manual__2023-04-08T23:11:34.257859+00:00 [queued]>
[2023-04-08T23:11:35.551+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: api_dag.create_table manual__2023-04-08T23:11:34.257859+00:00 [queued]>
[2023-04-08T23:11:35.552+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-04-08T23:11:35.552+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 2
[2023-04-08T23:11:35.553+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-04-08T23:11:35.564+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_table> on 2023-04-08 23:11:34.257859+00:00
[2023-04-08T23:11:35.571+0000] {standard_task_runner.py:55} INFO - Started process 6118 to run task
[2023-04-08T23:11:35.576+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'api_dag', 'create_table', 'manual__2023-04-08T23:11:34.257859+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/api.py', '--cfg-path', '/tmp/tmp6awp4qqw']
[2023-04-08T23:11:35.579+0000] {standard_task_runner.py:83} INFO - Job 16: Subtask create_table
[2023-04-08T23:11:35.661+0000] {task_command.py:376} INFO - Running <TaskInstance: api_dag.create_table manual__2023-04-08T23:11:34.257859+00:00 [running]> on host c12a518d534c
[2023-04-08T23:11:35.724+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=api_dag
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2023-04-08T23:11:34.257859+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-08T23:11:34.257859+00:00
[2023-04-08T23:11:35.737+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2023-04-08T23:11:35.743+0000] {sql.py:315} INFO - Running statement: CREATE TABLE IF NOT EXISTS user_details (
    firstname VARCHAR NOT NULL,
    lastname VARCHAR NOT NULL,
    country VARCHAR NOT NULL,
    username VARCHAR UNIQUE NOT NULL,
    password VARCHAR NOT NULL,
    email VARCHAR UNIQUE NOT NULL);, parameters: None
[2023-04-08T23:11:35.745+0000] {postgres.py:96} INFO - NOTICE:  relation "user_details" already exists, skipping

[2023-04-08T23:11:35.754+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=api_dag, task_id=create_table, execution_date=20230408T231134, start_date=20230408T231135, end_date=20230408T231135
[2023-04-08T23:11:35.790+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-04-08T23:11:35.811+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
